seastar/src/config_and_start_seastar.cc

```diff
@@ -0,0 +1,110 @@
+#include "config_and_start_seastar.hh"
+
+namespace seastar {
+
+// Copies rust::Vec<rust::String> as char**. Returns nullptr if fails.
+// Function free_args should be called on returned pointer to avoid memory leak.
+static char** args_as_ptr(const rust::Vec<rust::String>& args) {
+    char** av = (char**) calloc(args.size() + 1, sizeof(char*));
+    if (av == nullptr) {
+        return nullptr;
+    }
+
+    for (size_t i = 0; i < args.size(); i++) {
+        const rust::String& arg = args[i];
+        av[i] = (char*) calloc(arg.size() + 1, sizeof(char));
+        if (av[i] == nullptr) {
+            return nullptr;
```

Comment:
Oops, you are leaking memory here. What about the strings that you have already allocated and the `av` array?

Reply:
Yep, thanks.